src/components/UserMounts/EditUserMounts/EditUserMountsButton.js

```diff
@@ -25,14 +25,17 @@ export const EditUserMountsButton = (props) => {
   };
 
   const updateUserMounts = () => {
-    const userOwnedMounts = props.allUserMounts
-      .filter(
-        (userMount) =>
-          userMount.user_id === props.userId &&
-          userMount.expansion !== props.expansion &&
-          userMount.owned
-      )
-      .map((userMount) => userMount.mount_id);
+    const userOwnedMounts =
+      props.expansion === -1
```

Comment:
Consider replacing number literals in code with descriptive constants

Reply:
Replaced hard-coded numbers with constants.